### ISSUE TITLE
[CoE Starter Kit - BUG] ISSUE TITLE Duplicates created in CoE solution metadata table for Admin | Sync Template V3 (Connectors)

### ISSUE THREAD

**Issue by blanghorst** [microsoft/coe-starter-kit#8770]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

I've seen similar issues before but I didn't see a soluton for my particular case.  The Setup wizard hands at the "Run Setup Flows."  All the flows in the wizard finish except the Admin | Sync Template V3 (Connectors) flow.  When I look in Power Automate, Admin | Sync Template V3 (Connectors) did actually complete successfully.  When I looked at the CoE Solution Metadata table, there are 2 instances of Admin | Sync Template V3 (Connectors) - one marked as having run with a run date (LastRunPassed is checked), and the other has having not run.  I'm not quite sure how to handle this and I didn't find anything in search.  I tried checking LastRunPassed and adding a run date, but the REFRESH button on that page in the app is now gray.

### Expected Behavior

Setup wizard should detect flow run and proceed.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.33

### What app or flow are you having the issue with?

Admin | Sync Template V3 (Connectors)

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
This happens sometimes unfortunately. 

The best trick to get around it is to just give the second instance of the flow a last run date and pass. Then the setup wizard will not be blocked and the second instance of the flows will be cleaned up in about a week after the cleanup flows run.

**Comment by blanghorst** [comment]
Here's the crazy thing - I went in and gave the second flow a LastRunDate and a LastRunPassed as my first troubleshooting step.  It didn't seem to fix the issue but maybe I should've backed all the way to the initial step of the wizard and went through them all again.  

What I finally did was on the duplicate flow, I deleted "Inventory" from the CoESolutionArea since the logic on the gallery on that page seemed to be filtering on that property.  The duplicate row then seemed to disappear from Dataverse and I could then proceed with the wizard.
